spec.md

```diff
@@ -604,12 +605,12 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id/ser
 
 ### Response ###
 
-| Status Code  | Description  |
+| Status Code | Description |
 |---|---|
-| 201 Created  |  Binding has been created. The expected response body is below. |
-| 200 OK  |  MUST be returned if the binding already exists and the requested parameters are identical to the existing binding. The expected response body is below. |
-| 409 Conflict  | MUST be returned if the requested binding already exists. The expected response body is <code>{}</code>, though the description field MAY be used to return a user-facing error message, as described in [Broker Errors](#broker-errors).  |
-| 422 Unprocessable Entity  | MUST be returned if the broker requires that <code>app_guid</code> be included in the request body. The expected response body is: <code>{ "error": "RequiresApp", "description": "This service supports generation of credentials through binding an application only." }</code>  |
+| 200 OK | MUST be returned if the binding already exists and the requested parameters are identical to the existing binding. The expected response body is below. |
+| 201 Created | MUST be returned if the binding was created as a result of this request. The expected response body is below. |
+| 409 Conflict | MUST be returned if a service binding with the same id, for the same service instance, already exists but with different parameters. The expected response body is <code>{}</code>, though the description field MAY be used to return a user-facing error message, as described in [Broker Errors](#broker-errors). |
```

Comment:
Interested to hear what @shalako thinks on this point.

I think the spec was intending to give flexibility to brokers that may not be able to replay requests for some reason.